extensions/amp-accordion/1.0/accordion.jss.js

```diff
@@ -39,10 +39,16 @@ const header = {
 // or delete if not used
 const content = {};
 
+const contentHiddenMatchable = {
+  'content-visibility': 'hidden-matchable !important',
```

Comment:
Let's use `contentVisibility` in this format.

Reply:
Updated in latest revision.